app/routes/payment-requests.js

```diff
@@ -0,0 +1,51 @@
+const { post } = require('../api')
+const Joi = require('joi')
+const { schemeAdmin } = require('../auth/permissions')
+
+module.exports = [{
+  method: 'GET',
+  path: '/payment-request/reset',
+  options: {
+    auth: { scope: [schemeAdmin] },
+    handler: async (request, h) => {
+      return h.view('reset-payment-request')
```

Comment:
This brings up some v interesting questions as to how we name and structure our routes/views.

Do we have a defined way as to how this is done as a best practise? Comparing between this and `ffc-pay-request-editor` and even `ffc-sfi-apply-web` there seems to be some discrepancies.

Based on some repos this would be split into 2 files `reset` and `reset-success` under a shared folder of `payment-request` to give the full path of `payment-request/reset` as given. Should this be the standard? I understand that these paths need to be uniform between here and `ffc-pay-processing` but if we define a standard for all this would cause any naming conflicts.

I certainly think there is (in this e.g.) a greater call for the sep. of routes using a folder structure, i.e. have a folder named `payment-request` and under it would be `reset`, `reset-success` , etc -- this would be easier to read, find and manage.

Reply:
> Do we have a defined way as to how this is done as a best practise? Comparing between this and ffc-pay-request-editor and even ffc-sfi-apply-web there seems to be some discrepancies.

I would say `many` discrepancies`.  We did have a ticket on the SFI backlog to agree a set of standards built on top of the GDS API spec with our flavour of what we wanted to do.

Some work was drafted, but never got beyond the initial discussion.  This repo was also created several months before that.

In this PR, I tried to be consistent with what is already in the repo for this ticket (even though I don't agree with it, for example, `payment-schemes` and `payment-holds` could just be `schemes` and `holds`.  In some of the recent refactoring of this repo, the structure of folders has evolved a little bit to be more consistent with our work elsewhere, but not fully.  Particularly route and view names.

My suggestion would be that we add a ticket to the backlog to revisit the work we started on SFI to get some "rules" for our route setup, rather than try to agree it as part of this ticket.  Thoughts?

Reply:
Yeah, I completely agree with adding this as a ticket to be taken on in future sprints, my only question re: that is, "how do we ensure that we are actively flagging, defining and implementing changes to deal with this tech debt?"

My concern is that the development of new features is viewed as higher importance than this - to the the point - were addressing it is put off for the meantime with a TODO task attached to come back to it once the service is deployed and live in the Summer (but ofc we've just kicked the can down the road as integration with VetsVisits and others (development of new features) will once again take precedence) -- is this something we can regularly integrate with our sprint and sprint goals?

Reply:
What we used to do previously was ensure we always allocated a number of story points each sprint to pay back our tech debt.

Because of the short space of time we've had to deliver this service, we've not really been following that practice.  Look at the backlog, the majority of the critical features are now delivered so we have no excuses for not regularly paying back debt.  I'll bring it up on the retro that we should start doing this again and agree how we highlight debt on the board.  Previously we just put them under one epic so it was clear.